src/commands/cloud-run/flare.ts

```diff
@@ -204,6 +210,17 @@ export class CloudRunFlareCommand extends Command {
       }
     }
 
+    // Get recent revisions, which will be used to generate insights file
+    this.context.stdout.write(chalk.bold('\n🌧 Fetching recent revisions...\n'))
```

Comment:
What are revisions?

Reply:
<img width="646" alt="Screenshot 2023-08-11 at 1 27 03 PM" src="https://github.com/DataDog/datadog-ci/assets/29668820/22853746-d79c-4353-9695-e2244c7ffa85">

Revisions basically just keep track of every "version" of the cloud-run service. Whenever it is modified and redeployed, a new revision is created.
I talked to @IvanTopolcic and he mentioned that this could be useful to support. It also appears in the cloud-run UI (see image above).

I will also include the deployment time since deployment time is included on the cloud-run dashboard

<img width="532" alt="Screenshot 2023-08-11 at 1 45 35 PM" src="https://github.com/DataDog/datadog-ci/assets/29668820/c3aac068-4e1e-4422-acfa-d687c62de27d">

Reply:
Nice, thanks!